apps/prairielearn/src/pages/instructorAssessmentGroups/formRemoveMembers.ejs

```diff
@@ -1,12 +1,18 @@
 <form name="delete-member-form" method="POST">
-  UIDs:
-  <input type="text" class="form-control" placeholder="A@ex.com, B@ex.com" name="delete_member_uids">
-  <br>
+  <div class="form-group">
+    <label>UID:
+      <select class="form-control" name="user_id">
+        <% row.users.forEach((user) => { %>
+          <option value="<%= user.user_id %>"><%= user.uid %></option>
+        <% }); %>
+      </select>
```

Comment:
Nit: while nesting the `<select>` inside `<label>` is fine from an accessibility standpoint, we get better styling if we move it outside and add `class="form-label"` to the label. Make sure to wire up `id`/`for` if you make this change.

Reply:
AFAIK `form-label` does not exist in Bootstrap 4, are you sure? https://getbootstrap.com/docs/4.1/components/forms/

Reply:
Heh, you're right. Regardless, moving the `<select>` out of the `<label>` appears to be necessary for the enclosing `div.form-group` to do its thing correctly.

Reply:
Done in 581126b. Separating the label from the form control is enough to provide the improved UI.